Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - VariedXenoArtifactItem
+        - ClothingMaskArcadia
+        - ClothingShoesBootsMagCombatFilled
+        - WeaponLaserCannon
+        - ClothingEyesHudMedSec
+        - PowerCellMicroreactor
+        - MuteToxinChemistryBottle
+        - SmokeGrenade
+        - ClothingShoesBootsMagBlinding
+        - TechnologyDiskRare
+        - Truncheon
+        - TurboItemRechargerCircuitboard
+        - ResearchDisk10000
+        - MedicatedSuture
+        - RegenerativeMesh
+        - FoodPizzaArnold
+        - PinpointerUniversal
+      chance: 1
+
+- type: entity
+  id: RandomEventIllegalLoot 
+  name: Random Illegal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetsyndie
+    - type: RandomSpawner
+      prototypes:
+      - DnaScramblerImplanter
+      - CyberPen
+      - CigPackSyndicate
+      - WeaponRevolverPythonAP
+      - ClothingEyesGlassesOutlawGlasses
+      - SyndicatePersonalAI
+      - EnergySword
+      - WeaponPistolCobra
+      - EmpGrenade
+      - Stimpack
+      - ClusterGrenade
+      - EncryptionKeyBinary
+      - EnergyShield
+      - SyndieMiniBomb
+      - PenExplodingBox
+      - GrenadeShrapnel
+      - GrenadeIncendiary
+      - ExGrenade
+      - C4
+      - SmokeGrenade
+      - GrenadeFlashBang
+      - ClothingShoesBootsMagSyndie
+      - ClothingBackpackDuffelSyndicateEVABundle
+      - ClothingUniformJumpskirtOperative
+      - ClothingOuterVestWeb
+      - ClothingNeckScarfStripedSyndieGreen
+      - ClothingUniformJumpsuitOperative
+      - ClothingHandsGlovesBoxingRigged
+      - ScramImplanter
+      - StorageImplanter
+      - ClothingHeadHatOutlawHat
+      - ClothingBackpackDuffelSyndicateCostumeClown
+      - RubberStampSyndicate
+      - ClothingBackpackDuffelSyndicatePyjamaBundle
+      - ClothingBackpackDuffelSyndicateCarpSuit
+      - EnergyDagger
+      - RevolverCapGun
+      - BalloonSyn
+      - ClothingHeadHatCatEars
+      - RadioJammer
+      - JetpackBlackFilled
+      - SoapSyndie
+      - BoxEncryptionKeySyndie
+      - ClothingHeadsetAltSyndicate
+      - HappyHonkNukieSnacks
+      - NukeDiskFake
+      - LanternFlash
+      - CrateFunSyndicateSegway
+      - SlipocalypseClusterSoap
+      - CombatMedipen
+      - BriefcaseSyndieLobbyingBundleFilled
+      - MedkitCombatFilled
+      - Emag
+      - ClothingBackpackDuffelSyndicateBundleSamurai
+      - BorgModuleSyndicateWeapon
+      - ClothingOuterHardsuitSyndie
+      - ClothingOuterHardsuitSyndieMedic
+      - SyndiHypo 
+      - WeaponRifleAk
+      - WeaponRifleLecter
+      - WeaponPistolCHIMPUpgradeKit
+      - WeaponSniperHristov
+      - WeaponPistolN1984
+      - WeaponLauncherPirateCannon
+      - ClothingHeadHelmetSwatSyndicate
+      - ClothingShoesBootsWinterSyndicate
+      - ClothingBeltMilitaryWebbingMedFilled
+      - EnergyKatana
+      - PenCentcom
+      - FoodPizzaArnold
```

Comment:
Illegal?

Reply:
Its a pizza that has Omni in it. I was just trying to get filler in it also